TanoaPeakSimulator/TanoaPeakSimulator/Tanoa.py

```diff
@@ -0,0 +1,128 @@
+    # for chromosome in sequence_object_array:
+    #     for x in range(2000):
+    #         get_random_base_pair_string(location_length, bp_length, chromosome.sequence,
+    #                                     chromosome.bp_segment_arr)
+    # for chromosome in sequence_object_array:
+    #     chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+    #                                                                         bp_amount, chromosome.sequence)
+    chromosome = sequence_object_array[1]
+    for x in range(100):
+        get_random_base_pair_string(location_length, bp_length, chromosome.sequence, chromosome.bp_segment_arr)
+
+    chromosome.small_bp_segment_list = get_random_base_pair_string_list(chromosome.bp_segment_arr, bp_length,
+                                                                        bp_amount, chromosome.sequence)
+
+    generate_output_files(chromosome)
+
+
+# Gets a psudo-random string of a given amount of characters from the main sequence. This will move to taking
+# the buffers on the sides and saving that as well so when sampled later we don't have to return to the
+# Very large string. That's inefficient.
+# Parmeters: sequence_length: Length of the big sample that you want to take
+# bp_length: the length of a smaller base pair that you will sample from the broader region
+#   this is setup so that the buffer can be dynamic based on the size of the reads you want to match
+#   to the original chromosome are
+# Sequence: The actual character sequence, generally attached to a chromosome like 2L or 2R
+# array_of_segments: array of BasePairSegments (samples) from the original chromosome.
+#   This array is stored within a chromosome as bp_sequence_array and is modifying that object.
+def get_random_base_pair_string(sequence_length, bp_length, sequence, array_of_segments):
+    if len(sequence) > sequence_length + 101:  # Ignore sequences that don't have enough length to work...
+        random_start_location = random.randrange(99, len(sequence) - (sequence_length + 1))
+
+        # gets a random sample from the random start location to the sequence length. This eventually should be
+        # random_start_location + sequence_length and + buffer... This is a simple substring
+        mini_sequence = sequence[random_start_location: (random_start_location + sequence_length)]
+        # makes a BasePairSegment object from the mini_sequence string and associated data
+        temp_segment = BasePairSegment(random_start_location, mini_sequence, sequence_length, bp_length)
+        # stores that in the passed array from chromosome
+        array_of_segments.append(temp_segment)
+
+
+# Gets smaller reads from larger samples and returns them as a list
+# Parameters: list_of_bp_segments: the larger samples, stored as an array in a chromosome object
+# bp_length: the length of one single read. Generally between 50-100bps in length
```

Comment:
The specific numbers (e.g., 50-100bps) are super helpful to readers.  Good idea!